change/@microsoft-teams-js-51421b73-089f-43de-9076-75563e1f064c.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "none",
+  "comment": "Removing beta tag from Pages.CurrentApp Namespace",
```

Comment:
>Removing beta tag from Pages.CurrentApp Namespace [](http://example.com/codeflow?start=14&length=49)

To ensure the changelog is consistent across changes, please use past-tense and ensure that symbol names are in backticks.

Suggestion:

Removed beta tag from all functions in `pages.currentApp` namespace.

Reply:
```suggestion
  "comment": "Removed beta tag from `pages.currentApp` namespace",
```